### ISSUE TITLE
Missing output dependency information implies that output depends on all other scalar variables, is that intensional?

### ISSUE THREAD

**Issue by lausdahl** [controllab/fmi-export-20sim#7]
The model description which is exported will contain outputs in this format:

``` xml
<ModelStructure>
        <Outputs>
            <Unknown index="45" />
            <Unknown index="48" />
        </Outputs>
    </ModelStructure>
```

this implies that index 45 and 48 must be of type `output` which they are but also that they depend on all other scalar variables defined in the model. The FMI standard refers to this as the _worst case_ type of dependency. This is assumes if no dependency information is available.

Either the dependencies should be listed as `dependencies="1 2 3"` for dependency on `1`, `2` and `3` where the `dependenciesKind` can be used to detail the type of dependency.

Otherwise `dependencies=""` can be used to indicate no dependencies. 

See p. 61 in https://svn.modelica.org/fmi/branches/public/specifications/v2.0/FMI_for_ModelExchange_and_CoSimulation_v2.0.pdf

**Comment by margro** [comment]
Yes it is intentional, since we don't yet have the dependency information available during our code generation process.
This means that I cannot generate the dependencies="1 2 3" list right now.
I've created a 20-sim enhancement ticket for the code generation to export the dependency information such that we can generate the actual dependencies list.
